applications/Chat/evaluate/gpt_evaluate.py

```diff
@@ -0,0 +1,519 @@
+import concurrent.futures
+import os
+import random
+import re
+import time
+from copy import deepcopy
+from typing import Any, Dict, List
+
+import matplotlib.pyplot as plt
+import numpy as np
+import openai
+import pandas as pd
+import seaborn as sns
+import tqdm
+from utils import jdump, jload
```

Comment:
It seems that `utils` is internal package? 

If so, it should be the third group according the guideline: https://composed.blog/python/imports

Reply:
Ok, I will fix that.